active/ImageIcon/ImageIconSpec.md

```diff
@@ -0,0 +1,143 @@
+(so can be used where IconElement is required, such as 
+[NavigationViewItem.Icon](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.NavigationViewItem.Icon)).
+Other than the base class, ImageIcon is equivalent to Image.
+
+ImageIcon is a superset of the existing BitmapIcon minus the
+[BitmapIcon.ShowAsMonochrome](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon.ShowAsMonochrome)
+property, as it is not a requirement from design to have all icons be monochrome.
+Since ShowAsMonochrome will not be part of ImageIcon, the 
+[IconElement.Foreground](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement.Foreground)
+inherited from IconElement will not be suported in this control.
+It will be up to the application to update the icon to match the user's theme. 
+
+Along with the various IconElement types, which are elements, there are IconSource types.
+For example BitmapIcon and
+[BitmapIconSource](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIconSource).
+Similarly in this spec, along with `ImageIcon` is an `ImageIconSource`.
+
+In summary:
+
+```
+IconElement
+    BitmapIcon
+    ImageIcon **new
+
+IconSource
+    BitmapIconSource
+    ImageIconSource **new
+
+FrameworkElement
+    Image
+```
+
+# API Pages
+
+## ImageIcon class
+
+Represents an icon that uses an image as its content.
+
+ImageIcon can be used just like an
+[Image](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Image)
+element, except it derives from
+[IconElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement)
+and can therefore be used anywhere an IconElement is required, such as
+[NavigationViewItem.Icon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.NavigationViewItem.Icon).
+
+> Note: the ImageIcon.Foreground property is currently ignored.
+
+## Is this the right control? 
+
+Use an ImageIcon control when you want to display an image as your icon. Do not use an ImageIcon if you want to display an icon from a font. 
+
+## Examples
+Below is an example of how you can use ImageIcon in an App Bar Button Item.
+
+> Spec note: This example will also be added to the App Bar Button Item page with the rest of the icon controls. 
+
+XAML
+```xml
+<AppBarButton Label="ImageIcon" Click="AppBarButton_Click">
+    <AppBarButton.Icon>
+        <ImageIcon Source="Wheel.png"/>
+    </AppBarButton.Icon>
+</AppBarButton>
+```
+
+## ImageIcon member notes
+
+| Name	| Type | Description | Default |
+|:--- | :--- | :--- | :--- |
+|Source | ImageSource | Gets and sets the icon source file that will be used in the application. | Null |
+
+## ImageIconSource class
+
+_Same as above but with IconSource as the base type_
+
+# API Details
+
+```cs
+namespace Microsoft.UI.Xaml.Controls
+{
+    [webhosthidden]
+    unsealed runtimeclass ImageIcon : Windows.UI.Xaml.Controls.IconElement
+    {
+        ImageIcon();
+
+        Windows.UI.Xaml.Media.ImageSource Source{ get; set; };
```

Comment:
I expected to see cool `Image` stuff here, like `Stretch`, `ImageFailed`, etc. But the only thing from `Image` that showed up is the `ImageSource`.

Reply:
I wonder if this is the symptom of the name of the control and maybe it should be named something else that makes it clearer? Unfortunately I cannot think of a better name other than ImageIcon because you can add different image types that Image supports.

Reply:
I think the intro I added to the top might have confused this too by making this sound like the full `Image` element. Really the distinction this icon has vs `BitmapIcon` is that it supports vector graphics, so more than just bitmaps, and 'Image' is the word that means both. `ImageSource` is the existing mechanism for images, including the useful feature of stream support, so it's a useful means to the end.